commands/shell.go

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// ExecuteShell returns a cobra command to execute a line through the system shell.
+// This uses the os/exec package to execute the command. The default command name is `!`.
+func ExecuteShell() *cobra.Command {
+	shellCmd := &cobra.Command{
+		Use:                "!",
+		Short:              "Execute the remaining arguments with system shell",
+		DisableFlagParsing: true,
+		RunE: func(_ *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("command requires one or more arguments")
+			}
+
+			path, err := exec.LookPath(args[0])
+			if err != nil {
+				return err
+			}
+
+			shellCmd := exec.Command(path, args[1:]...)
```

Comment:
Won't it fail if there is only one argument?

Shouldn't the test on len look for 2, not 1

See https://github.com/reeflective/console/pull/58#discussion_r1844996446